frontend/src/components/staff/RequestTable.jsx

```diff
@@ -56,8 +57,24 @@ export default function RequestTable({ type, refresh, onRefresh }) {
     });
   }
 
+  if (type === 'unfulfilled') {
+    fulfilled = 0;
+  }
+  if (type === 'fulfilled') {
+    fulfilled = 1;
+  }
+  if (type === 'declined') {
+    fulfilled = -1;
+  }
+
   useEffect(() => {
-    fetch(`/api/get_${type}_requests`)
+    fetch(`/api/get_special_requests`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ "fulfilled": fulfilled }),
```

Comment:
nit (this might need a formatting fix due to eslint; we'll see when it tries to build)
```suggestion
      body: JSON.stringify({fulfilled}),
```